src/patient-registration/patient-registration.component.tsx

```diff
@@ -34,7 +41,7 @@ export interface FormValues {
   additionalFamilyName: string;
   addNameInLocalLanguage: boolean;
   gender: string;
-  birthdate: Date;
+  birthdate: string;
```

Comment:
Since these are initial form values, we don't actually want to use `Date` but rather a string of well formatted value as initial date value. The simplest way is to use a string, as for the date input component won't understand (parse) the date object. see: https://stackoverflow.com/a/6982754

Correct me if I'm wrong.